src/black/mode.py

```diff
@@ -168,35 +168,14 @@ def supports_feature(target_versions: Set[TargetVersion], feature: Feature) -> b
 class Preview(Enum):
     """Individual preview style features."""
 
-    add_trailing_comma_consistently = auto()
-    blank_line_after_nested_stub_class = auto()
-    blank_line_between_nested_and_def_stub_file = auto()
     hex_codes_in_unicode_sequences = auto()
```

Comment:
The changelog mentions that this preview style shipped but the preview-flag still exists.

Reply:
Oops, you're right. I must have confused myself somewhere. I guess this preview feature will have to wait another year.